docs/setup-guides/production-baremetal/quickstart.md

```diff
@@ -10,7 +10,7 @@ Commands in this section will use the system package manager, and run with root
 
 ```
 sudo apt update && sudo apt install -f -y \
-    python3 python3-virtualenv virtualenv \
+    python3 python3-virtualenv python3-venv virtualenv \
```

Comment:
I think you're right that we should use `python3-venv`. I think `python3-virtualenv` should actually be removed from here! In a Python 3 context `venv` is used as part of stdlib, but I guess Ubuntu doesn't ship that as part of the base python3 package for space reasons (?).